### PR TITLE
Add API overview page for SVG

### LINKED ISSUE #2646: Add API overview page for SVG
GroupData.json lists "SVG" as a Web API, but it doesn't have an API overview page. We should write a page and add the "overview" property to GroupData.

### PR DESCRIPTION
**Pull request by himanshugarg**
#### Summary
Adds API overview page for SVG

#### Motivation
Low hanging :mango:

#### Supporting details
Borrows heavily from [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model) and a bit from the [SVG Tutorial](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Getting_Started)

#### Related issues
Attempts to fix #2646

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/svg_api/index.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+- {{DOMxRef("SVGVKernElement")}} {{Deprecated_Inline}}
+
+### SVG data type interfaces
+
+Here are the DOM APIs for data types used in the definitions of SVG properties and attributes.
+
+#### Static type
+
+- {{DOMxRef("SVGAngle")}}
+- {{DOMxRef("SVGColor")}} {{Deprecated_Inline}}
```
Comment (estelle) [inline files/en-us/web/api/svg_api/index.md:126]: Do we want to include deprecated types (and interfaces) that have no documentation?

Comment (himanshugarg) [reply files/en-us/web/api/svg_api/index.md:126]: Thanks for the review. No we don't. Will remove.

File: files/en-us/web/api/svg_api/index.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  - API
+  - SVG
+  - Guide
+  - Overview
+  - Reference
+  - Web
+---
+{{DefaultAPISidebar("SVG")}}
+
+SVG provides elements for circles, rectangles, and simple and complex curves. The elements' attribute values specify how these must be drawn. The **SVG API** is the subset of the **DOM** connecting these SVG elements and their attribute values to scripts or programming languages by representing them in memory. The SVG API thus provides methods that allow programmatic access to the SVG Elements and their attribute values.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svg_api/index.md:14]:
```suggestion
SVG provides elements for circles, rectangles, and simple and complex curves. The elements' attribute values specify how these must be drawn. The **SVG API** is the subset of the **DOM** connecting these SVG elements and their attribute values to scripts or programming languages by representing them in memory. The SVG API thus provides methods that allow programmatic access to the SVG elements and their attribute values.
```

File: files/en-us/web/api/svg_api/index.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+The SVG API is a set of interfaces that have been categorized into the following broad categories:-
```
Comment (sideshowbarker) [inline files/en-us/web/api/svg_api/index.md:16]:
```suggestion
The SVG API is a set of interfaces that have been categorized into the following broad categories:
```

File: files/en-us/web/api/svg_api/index.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+1. [The Element interfaces](#svg_element_interfaces) provide access to the properties of SVG elements and methods to manipulate them.
+2. [The Static data type](#svg_data_type_interfaces) Interfaces provide access to element attribute values and methods to manipulate them.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svg_api/index.md:18]:
```suggestion
2. [The Static data type](#svg_data_type_interfaces) interfaces provide access to element attribute values and methods to manipulate them.
```

File: files/en-us/web/api/svg_api/index.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+3. For attributes that can be animated the [Animated data type interfaces](#svg_data_type_interfaces) provide read only access to the current animated value of an attribute.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svg_api/index.md:19]:
```suggestion
3. For attributes that can be animated, the [animated data type interfaces](#svg_data_type_interfaces) provide read only access to the current animated value of an attribute.
```

File: files/en-us/web/api/svg_api/index.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+  - Overview
+  - Reference
+  - Web
+---
+{{DefaultAPISidebar("SVG")}}
+
+SVG provides elements for circles, rectangles, and simple and complex curves. The elements' attribute values specify how these must be drawn. The **SVG API** is the subset of the **DOM** connecting these SVG elements and their attribute values to scripts or programming languages by representing them in memory. The SVG API thus provides methods that allow programmatic access to the SVG Elements and their attribute values.
+
+The SVG API is a set of interfaces that have been categorized into the following broad categories:-
+1. [The Element interfaces](#svg_element_interfaces) provide access to the properties of SVG elements and methods to manipulate them.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svg_api/index.md:17]:
```suggestion
1. [The element interfaces](#svg_element_interfaces) provide access to the properties of SVG elements and methods to manipulate them.
```

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/api/svg_api/index.md (hunk 1 above)
Comment (himanshugarg) [reply files/en-us/web/api/svg_api/index.md]: Removed. Thanks.

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**